schema/SystemStructureDescription.xsd

```diff
@@ -602,6 +602,45 @@
                                 </xs:documentation>
                             </xs:annotation>
                         </xs:group>
+                        <xs:element name="Dimension" minOccurs="0" maxOccurs="unbounded">
+                            <xs:annotation>
+                                <xs:documentation xml:lang="en">
+                                    This optional element specifies one dimension of an array connector.
+                                    If no dimension elements are present in a connector, it is a scalar
+                                    connector. The number of dimension elements in a connector provides
```

Comment:
defines

Reply:
@klausschuch can you explain why you prefer `defines`?